### PR TITLE
[1.1.x] Update Czech language

### PR DESCRIPTION
**Pull request by thinkyhead**
From #9657 by @petrzjunior

### CODE REVIEW

File: Marlin/language_cz.h [hunk 1]
```diff
@@ -348,10 +354,16 @@
 #define MSG_DRIVE_STRENGTH                  _UxGT("Buzeni motoru")
 #define MSG_DAC_PERCENT                     _UxGT("Motor %")
 #define MSG_DAC_EEPROM_WRITE                _UxGT("Ulozit do EEPROM")
+#define MSG_FILAMENT_CHANGE_HEADER_PAUSE    _UxGT("TISK POZASTAVEN")
+#define MSG_FILAMENT_CHANGE_HEADER_LOAD     _UxGT("ZAVEDENI FILAMENTU")
+#define MSG_FILAMENT_CHANGE_HEADER_UNLOAD   _UxGT("VYSUNUTI FILAMENTU")
+#define MSG_FILAMENT_CHANGE_OPTION_HEADER   _UxGT("MOŽNOSTI OBNOVENI:")
```
Comment (petrzjunior) [inline Marlin/language_cz.h]: I forgot a UTF char here, `MOZNOSTI OBNOVENI:` is correct.

File: Marlin/language_cz.h [hunk 2]
```diff
@@ -348,10 +354,16 @@
...
+#define MSG_FILAMENT_CHANGE_OPTION_PURGE    _UxGT("Vytlacit vic")
 #define MSG_FILAMENT_CHANGE_OPTION_RESUME   _UxGT("Obnovit tisk")
 #define MSG_FILAMENT_CHANGE_NOZZLE          _UxGT("  Tryska: ")
 #define MSG_ERR_HOMING_FAILED               _UxGT("Parkovani selhalo")
 #define MSG_ERR_PROBING_FAILED              _UxGT("Kalibrace selhala")
+#define MSG_M600_TOO_COLD                   _UxGT("M600: Moc studený")
```
Comment (petrzjunior) [inline Marlin/language_cz.h]: Same here, it should be `M600: Moc studeny`

**Comment by petrzjunior** [comment]
Sorry for my mistake, it should be corrected in 2.0.x branch as well. Shall I make an follow up?

**Comment by thinkyhead** [comment]
No problem, I'll take care of it.

**Pull request merged**